calico/getting-started/windows-calico/quickstart.md

```diff
@@ -376,6 +377,83 @@ The following steps install a Kubernetes cluster on a single Windows node, with
 %>
 {% endtabs %}
 
+Congratulations! You now have a Kubernetes cluster with {{site.prodnameWindows}} and a Linux control node.
+
+#### Install {{site.prodnameWindows}} using HostProcess containers
+
+With Kubernetes v1.22, a new Windows container type called "HostProcess containers" can run directly on the host with access to the host network namespace,
+storage and devices. With this feature, {{site.prodnameWindows}} can now be installed and managed using Kubernetes resources such as Daemonsets and ConfigMaps,
+instead of needing to configure and install {{site.prodnameWindows}} manually on each node. Using this installation method, the {{site.prodnameWindows}}
+services are no longer registered on the host. Instead, the services are run directly within HostProcess containers.
+
+> **Note**: This installation method is a tech preview and should not be used for production clusters. Upgrades from a tech preview version of this
+> installation method to the GA version might not be seamless.
+{: .alert .alert-info}
+
+##### Requirements
+
+In addition to the [{{site.prodnameWindows}} requirements]({{site.baseurl}}/getting-started/windows-calico/kubernetes/requirements),
+this installation method has [additional requirements](https://kubernetes.io/docs/tasks/configure-pod-container/create-hostprocess-pod/):
+
+- Kubernetes v1.22+
+- HostProcess containers support enabled: for v1.22, HostProcess containers support has to be [enabled](https://v1-22.docs.kubernetes.io/docs/tasks/configure-pod-container/create-hostprocess-pod/#before-you-begin-version-check). For Kubernetes v1.23+, HostProcess containers are enabled by default.
```

Comment:
Same here...use external link

{% include open-new-window.html text='enabled' url='[URL](https://v1-22.docs.kubernetes.io/docs/tasks/configure-pod-container/create-hostprocess-pod/#before-you-begin-version-check)' %}